packages/@aws-cdk/aws-appmesh/lib/virtual-router.ts

```diff
@@ -184,23 +177,24 @@ export class VirtualRouter extends VirtualRouterBase {
 export interface VirtualRouterAttributes {
   /**
    * The name of the VirtualRouter
+   *
+   * @default - required if virtualRouterArn is not specified
    */
   readonly virtualRouterName?: string;
 
   /**
    * The Amazon Resource Name (ARN) for the VirtualRouter
+   *
+   * @default - required if virtualRouterName and mesh are not specified
    */
   readonly virtualRouterArn?: string;
```

Comment:
Same comment here (remove `virtualRouterArn` from `VirtualRouterAttributes`, make `virtualRouterName` and `mesh` required).